Dockerfile

```
@@ -37,14 +46,88 @@ RUN git clone --depth=1 --single-branch --branch=frankenphp-8.2 https://github.c
         --enable-embed=static \
         --enable-zts \
         --disable-zend-signals \
-        --with-config-file-scan-dir=/usr/local/lib/php/ && \
+    	# --enable-mysqlnd is included here because it's harder to compile after the fact than extensions are (since it's a plugin for several extensions, not an extension in itself)
+    	--enable-mysqlnd \
+     	# make sure invalid --configure-flags are fatal errors instead of just warnings
+    	--enable-option-checking=fatal \
+    	# https://github.com/docker-library/php/issues/439
+    	--with-mhash \
+    	# https://github.com/docker-library/php/issues/822
+    	--with-pic \
+    	# --enable-ftp is included here because ftp_ssl_connect() needs ftp to be compiled statically (see https://github.com/docker-library/php/issues/236)
+    	--enable-ftp \
+    	# --enable-mbstring is included here because otherwise there's no way to get pecl to use it properly (see https://github.com/docker-library/php/issues/195)
+    	--enable-mbstring \
+    	# https://wiki.php.net/rfc/argon2_password_hash
+    	--with-password-argon2 \
+    	# https://wiki.php.net/rfc/libsodium
+    	--with-sodium=shared \
+    	# always build against system sqlite3 (https://github.com/php/php-src/commit/6083a387a81dbbd66d6316a3a12a63f06d5f7109)
+		--with-pdo-sqlite=/usr \
+		--with-sqlite3=/usr \
+		--with-curl \
+		--with-iconv \
+		--with-openssl \
+		--with-readline \
+		--with-zlib \
+    	# https://github.com/bwoebi/phpdbg-docs/issues/1#issuecomment-163872806 ("phpdbg is primarily a CLI debugger, and is not suitable for debugging an fpm stack.")
+		--disable-phpdbg \
+        --with-config-file-path="$PHP_INI_DIR" \
+        --with-config-file-scan-dir="$PHP_INI_DIR/conf.d" && \
     make -j$(nproc) && \
     make install && \
     rm -Rf php-src/ && \
     ldconfig && \
+    install-php-extensions opcache && \
     php --version
 
-RUN echo "zend_extension=opcache.so\nopcache.enable=1" > /usr/local/lib/php.ini
+#RUN echo "zend_extension=opcache.so\nopcache.enable=1" > /usr/local/lib/php.ini
+
+FROM golang AS server
```

```suggestion
FROM golang AS builder
```

`builder` is more relevant IMHO